infra/bisector.py

```diff
@@ -47,7 +48,7 @@ class BuildData():
   Attributes:
     project_name: The name of the OSS-Fuzz project that is being checked
     engine: The fuzzing engine to be used
-    sanitizer: The fuzzing sanitizer to be used
+    sanitizer: The sanitizer to be used
     architecture: The system architecture being fuzzed
```

Comment:
The explaination of architechture isn't super helpful. Maybe the "CPU architecture to build the fuzzer for"